pages/basic_concepts/promotion.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Promotion policies
+last_updated: January 11, 2023
+toc: false
+permalink: promotion.html
+---
+
+## What is promotion policy
+Promotion policy is a "filter" working on per-request basis that determines
+what gets inserted(promoted) into the cache. This filter is meant to limit cache
+pollution resulting from insertion of rarely accessed data (e.g. I/Os resulting
+from Ceph scrub operation). If promotion policy rejects a request it is
+handled in pass-through, similarly to what happens when request couldn't be inserted
```

Comment:
*handled in pass-through *mode* 

Suggestion: link "pass-through" to the PT engine description?